include/ParameterSet.hpp

```diff
@@ -409,6 +411,79 @@ class ParameterSet {
     }
 
     /**
+     * Add a single GeographyPoint.
+     */
+    ParameterSet& addGeographyPoint(const GeographyPoint &val) {
+        validateType(WIRE_TYPE_GEOGRAPHY_POINT, false);
+        m_buffer.ensureRemaining(1 + 2*sizeof(double));
```

Comment:
This magic number should be a constant from GeographyPoint, no?  Or even call a method of GeographyPoint to do the serializing.